test_unbounded_knapsack.py

```diff
@@ -0,0 +1,49 @@
+"""
+Created on Tue Aug 31 16:31:07 2022
+
+This file contains the test-suite for the unbounded_knapsack problem.
+"""
+import unittest
+
+from unbounded_knapsack import unbounded_knapsack as k

```

Comment:
Directly import the function we need...
```suggestion
from from unbounded_knapsack.unbounded_knapsack import unbounded_knapsack
```